apps/src/StudioApp.js

```diff
@@ -15,8 +15,8 @@ var FeedbackUtils = require('./feedback');
 * The minimum width of a playable whole blockly game.
 */
 var MIN_WIDTH = 900;
-var MIN_MOBILE_SHARE_WIDTH = 450;
-var MOBILE_NO_PADDING_SHARE_WIDTH = 400;
+var MOBILE_SHARE_WIDTH_PADDING = 50;
```

Comment:
Instead of having two constants (min width = 450, min no padding width = 400), I switched to have one constant DEFAULT_MIN_NO_PADDING_SHARE_WIDTH = 400 and then added MOBILE_SHARE_WIDTH_PADDING = 50.

This way you can add 50 to 400 (or whatever other value) instead of hardcoding both constants.